rpcs3/Emu/Cell/Modules/sys_net.cpp

```diff
@@ -193,7 +193,7 @@ namespace sys_net
 #endif
 		{
 			ERROR_CASE(EWOULDBLOCK);
-		default: throw fmt::exception("Unknown/illegal socket error: %d" HERE, code);
+		default: libnet.error("Unknown/illegal socket error: %d" HERE, code);
```

Comment:
Is this fully analogous? Will it pause the emulations (not sure if it's worth it though)